ui/src/components/TestRunner/index.tsx

```diff
@@ -94,7 +88,10 @@ export const TestRunner = () => {
       sepGroup.tests.push(testRun);
     }
     return groupedTestRuns;
-  }
+  };
+
+  const validateFormData = () =>
+    !!formData.seps.length && !!formData.homeDomain;
```

Comment:
we now check to make sure we have all necessary info before showing the "Run Test" btn